query_generators/anthropic.py

```diff
@@ -110,16 +114,43 @@ def generate_query(
                     columns_to_keep,
                     shuffle,
                 )
+                pruned_metadata_ddl = convert_postgres_ddl_to_dialect(
```

Comment:
sorry forgot to highlight earlier. But i rmb adding this extra line before the translation in gen_prompt.py:
```
# remove triple backticks
 pruned_metadata_ddl = pruned_metadata_ddl.replace("```", "").strip()
```
because sqlglot can't translate with those. would it also be a problem here and in the openai script?

Reply:
Thanks for pointing this! Hasn't been a problem - sqlite evals for OpenAI and Anthropic both worked super smoothly :D